docs/deployment-resource-isolation.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: global
+displayTitle: Gearpump Resource Isolation
+title: Gearpump Resource Isolation
+description: Gearpump Resource Isolation
+---
+
+cgroups (abbreviated from control groups) is a Linux kernel feature to limit, account, and isolate resource usage (CPU, memory, disk I/O, etc.) of process groups.
+
+In Gearpump, we use cgroup to manage CPU resources. You need to check the following items and configuration before use.
+
+*  Check current user's uid and gid in /etc/passwd file. For example if user is gear, then check gear's uid and gid in /etc/passwd file
+*  Cgroups feature is supported or not by current system's kernel
+
+   Check /etc/cgconfig.conf exist or not. If not exists, please "yum install libcgroup". If exists, set CPU sub system's mount directory location, and change uid/gid to Gearpump user's uid/gid in that config file, let's take 500 for example, pay attention to the uid/gid mentioned before.
```

Comment:
"let's take 500 for example, pay attention to the uid/gid mentioned before."

What is 500?